specification/servicefabric/data-plane/Microsoft.ServiceFabric/6.0/servicefabric.json

```diff
@@ -0,0 +1,12826 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Service Fabric Client APIs",
+    "description": "Service Fabric REST Client APIs allows management of Service Fabric clusters, applications and services.",
+    "version": "6.0"
+  },
+  "host": "localhost:19080",
```

Comment:
Please fix this

Reply:
What's wrong with using localhost here?

Reply:
This will be the default Url for the client when the user doesn't specify one.

Reply:
That seems like okay behaviour, if people are using the Service Fabric SDK they will have a server running at that URL.

Reply:
The Service Fabric client APIs can be pointed at any clusters, there is no fixed URI for the service so we are using the development cluster URI. We will not fixing this. Thanks.

Reply:
@samedder Which server? AutoRest generated clients won't spin off a server.
@VipulM-MSFT If your host is parametrized - there's a template of the cluster URL, then you should use `x-ms-parametrized-host` like this example: https://github.com/Azure/azure-rest-api-specs/blob/current/specification/cognitiveservices/data-plane/TextAnalytics/v2.0/TextAnalytics.json#L14. Even if the entire URl is up to the user, you should follow examples like this: https://github.com/Azure/azure-rest-api-specs/blob/current/specification/keyvault/data-plane/Microsoft.KeyVault/2016-10-01/keyvault.json#L9.

Reply:
@jianghaolu - this is correctly pointing to the local development cluster that ships with the SDK. We will not be changing this for this release. This is not a change in this release and we had already discussed this with the Swagger team as Microsoft to ensure that we are following the right practices. We are open for discussion on the next revision but not this one.